ui/src/components/chat/compose/chatbar.rs

```diff
@@ -274,117 +274,161 @@ pub fn get_chatbar<'a>(cx: &'a Scoped<'a, super::ComposeProps>) -> Element<'a> {
         .collect::<Vec<_>>();
 
     let disabled = !state.read().can_use_active_chat();
+    let error = use_state(cx, || (false, id));
+    let value_chatbar = state
+        .read()
+        .get_active_chat()
+        .as_ref()
+        .and_then(|d| d.draft.clone())
+        .unwrap_or_default();
 
-    let chatbar = cx.render(rsx!(Chatbar {
-        key: "{id}",
-        id: id.to_string(),
-        loading: is_loading,
-        placeholder: get_local_text("messages.say-something-placeholder"),
-        is_disabled: disabled,
-        ignore_focus: cx.props.ignore_focus,
-        onchange: move |v: String| {
-            if let Some(id) = &active_chat_id {
-                state.write_silent().mutate(Action::SetChatDraft(*id, v));
-                update_send();
-                local_typing_ch.send(TypingIndicator::Typing(*id));
-            }
-        },
-        value: state
+    if value_chatbar.len() >= 1024 && !error.0 {
```

Comment:
Why don't we use that default prop above (which is in another file but still) so if we want to change it later we can change it in just one place?